esphome/wizard.py

```diff
@@ -136,58 +138,61 @@ def wizard_file(**kwargs):
 
     config += HARDWARE_BASE_CONFIGS[kwargs["platform"]].format(**kwargs)
 
-    config += LOGGER_API_CONFIG
-
-    # Configure API
-    if "password" in kwargs:
-        config += f"  password: \"{kwargs['password']}\"\n"
-    if "api_encryption_key" in kwargs:
-        config += f"  encryption:\n    key: \"{kwargs['api_encryption_key']}\"\n"
-
-    # Configure OTA
-    config += "\nota:\n"
-    if "ota_password" in kwargs:
-        config += f"  password: \"{kwargs['ota_password']}\""
-    elif "password" in kwargs:
-        config += f"  password: \"{kwargs['password']}\""
-
-    # Configuring wifi
-    config += "\n\nwifi:\n"
-
-    if "ssid" in kwargs:
-        if kwargs["ssid"].startswith("!secret"):
-            template = "  ssid: {ssid}\n  password: {psk}\n"
+    config += LOGGER_CONFIG
+
+    if kwargs["board"] != "rpipico":
+        config += API_CONFIG
```

Comment:
Its much better to return early instead of putting everything inside a conditional branch
```suggestion
    if kwargs["board"] == "rpipico":
    	return config
    	
    config += API_CONFIG
```